Formula/php-cs-fixer.rb

```diff
@@ -0,0 +1,28 @@
+class PhpCsFixer < Formula
+  desc "Tool to automatically fix PHP coding standards issues"
+  homepage "https://wp-cli.org/"
+  url "https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v2.11.1/php-cs-fixer.phar"
+  version "2.11.1"
+  sha256 "cb1092637760c7283f63af3ef74ae835a861b123f8592a0aa195bd757cd1e088"
+
+  bottle :unneeded
+
+  depends_on "php" if MacOS.version <= :el_capitan
+
+  def install
+    bin.install "php-cs-fixer.phar" => "php-cs-fixer"
+  end
+
+  test do
+    (testpath/"test.php").write <<~EOS
+      <?php $a = new    class(2){};
```

Comment:
Can't use anonymous classes with PHP 5.6, which is what Sierra is picking up by default.

Reply:
Thanks kabel, I used something "simpler" now.